crates/topos-tce/src/app_context/network.rs

```diff
@@ -42,11 +42,21 @@ impl AppContext {
                         );
 
                         match self.validator_store.insert_pending_certificate(&cert) {
-                            Ok(Some(_)) => {
+                            Ok(Some(pending_id)) => {
                                 debug!(
                                     "Certificate {} has been inserted into pending pool",
                                     cert.id
                                 );
+
+                                _ = self
+                                    .tce_cli
+                                    .get_double_echo_channel()
+                                    .send(DoubleEchoCommand::Broadcast {
+                                        need_gossip: false,
+                                        cert,
+                                        pending_id,
```

Comment:
The `pending_id` is actually the same as `cert.id` right? Why include the same ID twice in the same message? Isn't it better to just have a bool then? `is_pending`?

Reply:
`pending_id` is not the same as `cert.id`, the `pending_id` is the place of the `certificate` in the `pending_pool`.
It is a gapless always incrementing `u64` where the `cert.id` is a `[u8;32]`